src/golang/lib/database/stmt_preparers/update_execution_state.go

```diff
@@ -0,0 +1,44 @@
+package stmt_preparers
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/aqueducthq/aqueduct/lib/models/shared"
+)
+
+func (s *StandardPreparer) PrepareUpdateExecStateStmt(
+	columnAccessPath string,
+	status shared.ExecutionStatus,
+	timestamp time.Time,
+	offset int,
+) (fragment string, args []interface{}, err error) {
+	timestampField, err := shared.ExecutionTimestampsJsonFieldByStatus(status)
+	if err != nil {
+		return "", nil, err
+	}
+
+	// (likawind) If passing the timestamp object directly to query, it somehow bypass the
```

Comment:
hmm thats interesting because we insert `time.Time` objects directly elsewhere in our codebase.

Reply:
I think it's because we insert it as a part of `json_set` and it somehow doesn't work 😭

Reply:
Elsewhere we mainly insert directly to the column